yt_dlp/__init__.py

```diff
@@ -62,6 +62,7 @@
     traverse_obj,
     variadic,
     write_string,
+    Config,
```

Comment:
this needs to be moved up with the classes

Reply:
Done